recipes/kpop/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+if [ "$(uname)" == "Darwin" ]; then
+    echo "Installing KPop for OSX."
+else 
+    echo "Installing KPop for Linux"
+fi
```

Comment:
is this required for anything? Otherwise it's understood (via the way packages work) that the installation is specific to platform.

Reply:
No, you are right, this is not needed and can be removed